components/compliance-service/ingest/projectupdater/manager.go

```diff
@@ -12,6 +12,7 @@ import (
 
 	iam_v2 "github.com/chef/automate/api/interservice/authz/v2"
 	automate_event "github.com/chef/automate/api/interservice/event"
+	"github.com/chef/automate/components/compliance-service/config"
```

Comment:
These are the same changes as the ingest-service's project update manager. 
* The `stage` is only updated by a single thread. 
* The `stage` is back up to a file through the `config.ConfigManager`